examples/src/test/java/com/opengamma/strata/examples/marketdata/ExampleMarketDataBuilderTest.java

```diff
@@ -242,14 +243,14 @@ private void assertBuilder(ExampleMarketDataBuilder builder) {
     assertEquals(MARKET_DATA_DATE, snapshot.getValuationDate().getSingleValue());
 
     for (ObservableId id : TIME_SERIES) {
-      assertTrue(snapshot.containsTimeSeries(id), "Time-series not found: " + id);
+      assertFalse(snapshot.getTimeSeries(id).isEmpty(), "Time-series not found: " + id);
```

Comment:
We gave up the ability to distinguish that when we changed `getTimeSeries` to return an empty series for an unknown ID. That was a deliberate decision. Having `containsTimeSeries` makes no sense after that.